DigitalLearningSolutions.Web/Models/DelegateRegistrationData.cs

```diff
@@ -1,16 +1,23 @@
 ﻿namespace DigitalLearningSolutions.Web.Models
 {
+    using System;
     using DigitalLearningSolutions.Web.ViewModels.Register;
 
     public class DelegateRegistrationData : RegistrationData
     {
         public DelegateRegistrationData() { }
 
-        public DelegateRegistrationData(int? centreId, int? supervisorDelegateId = null, string? primaryEmail = null) : base(centreId)
+        public DelegateRegistrationData(
+            int? centreId,
+            int? supervisorDelegateId = null,
+            string? primaryEmail = null,
+            DateTime? primaryEmailVerified = null

```

Comment:
This doesn't look like it's every getting used...

Reply:
removed